docs/framework/deployment/deployment-guide-for-developers.md

```diff
@@ -137,6 +139,10 @@ If you use ClickOnce, InstallShield, or WiX to deploy your app, you can add a de
 
 8.  In the **Property Pages** dialog box, choose **OK**.
 
+<a name="installaware"></a> 
+### InstallAware deployment
+InstallAware builds Windows app (APPX), Windows Installer (MSI), Native Code (EXE), and App-V (Application Virtualization) packages from a single source. Easily [include any version of the .NET Framework](https://www.installaware.com/one-click-pre-requisite-installer.htm) in your setup, optionally customizing the installation by [editing the default scripts](https://www.installaware.com/msicode.htm) (for example, InstallAware pre-installs certificates on Windows 7, without which the .NET 4.7 setup fails). For more information on InstallAware, see the [InstallAware for Windows Installer](https://www.installaware.com/) website.
```

Comment:
remove the for example sentence from the parentheses and put it in a new sentence. Such as:
... in your setup, optionally customizing the installation by [editing the default scripts](https://www.installaware.com/msicode.htm). For example, InstallAware pre-installs certificates on Windows 7, without which the .NET 4.7 setup fails.

Also change .NET 4.7 to .NET Framework 4.7.